vscode-test-runner/workspace-1/.joyride/src/integration_test/nrepl_eval_test.cljs

```diff
@@ -0,0 +1,69 @@
+(ns integration-test.nrepl-eval-test
+  (:require [cljs.test :refer [testing is use-fixtures async]]
+            [integration-test.macros :refer [deftest-async]]
+            ["nrepl-client" :as nrepl-client]
+            [promesa.core :as p]
+            ["vscode" :as vscode]
+            ["util" :as util]))
+
+(defn- from-nrepl-messages [messages k]
+  (.reduce messages
+           (fn [answer message]
+             (if (aget message k)
+               (str answer (aget message k))
+               answer))
+           ""))
+
+(use-fixtures :once
+  {:before
+   #(async done 
+           (p/do!
+            (p/create (fn [resolve reject]
+                        (p/let [port (vscode/commands.executeCommand "joyride.startNReplServer")
+                                client' (.connect nrepl-client  #js {:port port})
+                                eval' (util/promisify client'.eval)]
+                          (def client client')
```

Comment:
some inline defs here

Reply:
It's on purpose. The tests use the them.

Reply:
Can't we use an atom for this? Using an inline def like this feels like a hack.

Reply:
Sure. Not that I see what's so hacky with using the namespace for it. 😄 But an atoms works too.